### PR TITLE
docs: Correct the base path for Admin REST APIs.

### LINKED ISSUE #11007: The base path of admin API is not correct in doc
### Describe the bug

The base path of admin API is not correct.

image

### Version

17.0

### Expected behavior

The base path for Admin API should be /auth/admin/realms.

Only change the base url to /auth/admin/realms you can get the correct result, for example:

image

### Actual behavior

The docs shows the base path as /auth.

By composing the url you get endpoint 404 not found, for example:

image

### How to Reproduce?

See https://www.keycloak.org/docs-api/17.0/rest-api/#_role_mapper_resource

### Anything else?

fix: https://github.com/keycloak/keycloak/pull/10933

### PR DESCRIPTION
**Pull request by Jeff-Tian**
Closes #11007 .

image

### CODE REVIEW

**Comment by Jeff-Tian** [comment]
Hi @ahus1 , thanks for checking it, now both the commit message & PR description are updated.

**Comment by andymunro** [comment]
Thanks for asking @ahus1.   Actually I think @hmlnarik or @pedroigor could better answer that question.

File: services/src/docs/asciidoc/overview.adoc [hunk 1]
```diff
@@ -8,5 +8,5 @@ Version: 1
 
 === URI scheme
 Host: localhost:8080
-BasePath: /auth
+BasePath: /auth/admin/realms
```
Comment (jonkoops) [inline services/src/docs/asciidoc/overview.adoc:11]: Isn't this as follows under Quarkus?
```suggestion
BasePath: /admin/realms
```

Comment (ahus1) [reply services/src/docs/asciidoc/overview.adoc:11]: That's true with Quarkus, but squeezing out all `auth` paths in all of Keycloak would be a different thing I believe. I was  willing to approve it as is.

Comment (jonkoops) [reply services/src/docs/asciidoc/overview.adoc:11]: Do we have an issue logged for this though? And if not, can you create one?

Comment (ahus1) [reply services/src/docs/asciidoc/overview.adoc:11]: Now we have an issue: #11256

Comment (jonkoops) [reply services/src/docs/asciidoc/overview.adoc:11]: Nice, consider this resolved then!

**Review by jonkoops — APPROVED** [review]

**Review by ahus1 — APPROVED** [review]

**Pull request merged**